README.md

```diff
@@ -1 +1,159 @@
-# Recipe-App
\ No newline at end of file
+<div align= 'center'>
+<a name="readme-top"></a>
+
+  <h3><b>**Recipe App**</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Recipe App] <a name="The Recipe App is a digital platform designed to simplify and enhance the cooking experience for users of all levels. This application provides a user-friendly interface for creating, sharing, and discovering a wide variety of recipes"></a>
+
+
+**[Recipe App]** is based on Ruby Language and Rails framework.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href=#>Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href=#>Localhost</a></li>
+  </ul>
```

Comment:
- [ ] Kindly fix this two links that redirected to a wrong page, doing this will make your README file to look professional. 👍